publish-to-maven-central/properties.sh

```diff
@@ -14,8 +14,8 @@
 
 # ECLIPSE:
 DROPS4=/home/data/httpd/download.eclipse.org/eclipse/downloads/drops4
-SDK_BUILD_DIR=R-4.26-202211231800
-SDK_VERSION=4.26
+SDK_BUILD_DIR=R-4.27-202303111800
```

Comment:
This will have to be the the correct final 4.27 build location which we won't know until tomorrow at the soonest.